src/player_team.ts

```diff
@@ -1125,15 +1140,16 @@ class Team {
       }
 
       let val = ping.damage;
-      val = Math.fround(val) * Math.fround(partialAtk(leadId, ping, healing) * 100) / Math.fround(100);
-      val = Math.fround(val) * Math.fround(partialAtk(helpId, ping, healing) * 100) / Math.fround(100);
+      val = Math.fround(val) * Math.fround(partialAtk(leadId, ping, healingFromCombos) * 100) / Math.fround(100);
+      val = Math.fround(val) * Math.fround(partialAtk(helpId, ping, healingFromCombos) * 100) / Math.fround(100);
       mults[i].badge = atkBadgeMult;
-      mults[i].lead = partialAtk(leadId, ping, healing);
-      mults[i].help = partialAtk(helpId, ping, healing);
+      mults[i].lead = partialAtk(leadId, ping, healingFromCombos);
+      mults[i].help = partialAtk(helpId, ping, healingFromCombos);
       ping.damage = Math.round(val * atkBadgeMult);
       mults[i].final = ping.damage;
     }
 
+    let healing = -poison + healingFromCombos;
     healing += this.countAwakening(Awakening.AUTOHEAL) * 1000;
```

Comment:
Combine with the above line and make it a const.

Reply:
Can do. Do you have preferences on line length? I prefer to keep lines around 80 characters but it looks like there's a good number of long lines in the codebase already.